sdk/storage/azblob/blockblob/client.go

```diff
@@ -27,39 +27,44 @@ import (
 	"github.com/Azure/azure-sdk-for-go/sdk/storage/azblob/internal/shared"
 )
 
+// ClientOptions contains the optional parameters when creating a Client.
+type ClientOptions struct {
+	azcore.ClientOptions
```

Comment:
So for _input_ types would we prefer to inline all the options rather than embed? Or perhaps give it a name of 'Core' or something so it's a bit easier to initialize?

Reply:
Client options is a bit of a special case.  We want to be able to add a new policy and its options type in `azcore` only without requiring an update of SDKs.

Reply:
Seems sensible to me.